src/components/Arrow/Arrow.tsx

```diff
@@ -30,7 +30,7 @@ const Arrow: FC<ArrowProps> = ({ edge, edgeName, widthOffset, currentNode }) =>
       <ArrowPath points={edge.points} arrowheadId={arrowheadId} widthOffset={widthOffset} />
       {label ? (
         <text x={label.x + widthOffset} y={label.y}>
-          {label.text}
+          {label.text.length > 12 ? label.text.substring(0, 11) + '...' : label.text}
```

Comment:
This works but from a user standpoint it may be frustrating when the screen is large enough to accommodate  more than 12 characters. Perhaps make it a function of the screen size? 
![Screen Shot 2020-07-17 at 2 56 15 PM](https://user-images.githubusercontent.com/53485517/87821327-ab41d500-c83d-11ea-9ca6-c2c2d0741650.png)

Reply:
Is there a way to include the full text as a tooltip? I'm always frustrated by truncated text that doesn't display the full value when I hover over it.

Reply:
Added a tooltip for labels that are truncated.

Reply:
Dang that tooltip is fresh
![image](https://user-images.githubusercontent.com/13512036/88181563-0c322a00-cbfd-11ea-84cc-6b1582853964.png)

I definitely agree it would be nice to expand the text if there is space, but there are probably a lot of cases we'd need to take into account (to prevent overlapping) and that could get complicated. @blangley28 thoughts?

Reply:
Yea I think that would be really nice to have but out of scope here. Tooltip is really nice but can we make it a bit bigger? I'm not sure if it needs to be as large as the original text but its hard for me to read at first glance

Reply:
@blangley28 confirmed this was his last comment so this is good to merge now